src/api/cvc4cpp.cpp

```diff
@@ -1080,6 +1081,7 @@ Term Term::notTerm() const
 
 Term Term::andTerm(const Term& t) const
 {
+  CVC4_API_CHECK_NOT_NULL;
```

Comment:
Maybe we should also check `t` for being null? What do you think?

Reply:
Good point, will add.

Reply:
Done, PTAL